packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -135,6 +140,12 @@ contract DeployConfig is Script {
         preimageOracleMinProposalSize = stdJson.readUint(_json, "$.preimageOracleMinProposalSize");
         preimageOracleChallengePeriod = stdJson.readUint(_json, "$.preimageOracleChallengePeriod");
         preimageOracleCancunActivationTimestamp = stdJson.readUint(_json, "$.preimageOracleCancunActivationTimestamp");
+
```

Comment:
It is dangerous to use non useful default values.

I have been concerned about how easy it would be to accidentally shoot self in foot here because this would pass the checks but result in a broken system

Longer term we will need to log the values used in the init calls so that we can have more legibility into what values are used

Reply:
Can you just update these to be sane values instead of 0?